docs/doc/13-sql-reference/10-data-types/42-data-type-variant.md

```diff
@@ -232,4 +224,4 @@ SELECT sum(arr[0]::INT) FROM array_table GROUP BY arr[0]::INT;
 
 ## Functions
 
-See [Semi-structured Functions](/doc/reference/functions/semi-structured-functions).
\ No newline at end of file
+See [Variant Functions](/doc/reference/functions/semi-structured-functions).
```

Comment:
Exhaustive list of all broken links found:
--
00:23:13.115 |  
00:23:13.115 | - On source page path = /doc/sql-commands/ddl/table/ddl-create-table:
00:23:13.115 | -> linking to /doc/sql-reference/data-types/data-type-semi-structured-types
00:23:13.115 |  
00:23:13.115 | - On source page path = /doc/sql-commands/query-syntax/dml-json-path:
00:23:13.116 | -> linking to /doc/sql-reference/data-types/data-type-semi-structured-types
00:23:13.116 |  
00:23:13.116 | - On source page path = /doc/sql-reference/data-types/data-type-variant:
00:23:13.116 | -> linking to /doc/reference/functions/semi-structured-functions